src/components/header/header.css

```diff
@@ -16,3 +16,7 @@
 .material-icons{
     font-size: 24px !important;
 }
+
+.menu {
+    right: -16px !important;
```

Comment:
`menu` is a theme param, and it passes to the component and merges with other styles. it is so hard to style `Menu` component through `IconMenu` component.